app/src/main/java/com/amaze/filemanager/database/UtilitiesDatabase.java

```diff
@@ -328,6 +328,21 @@ public void migrate(@NonNull SupportSQLiteDatabase database) {
         }
       };
 
+  private static final Migration MIGRATION_4_5 =
+      new Migration(4, 5) {
+        @Override
+        public void migrate(@NonNull SupportSQLiteDatabase database) {
+          database.execSQL(
```

Comment:
What happens to the old table, that could have indexes repeated

Reply:
And hell, I was even indexing the wrong table :(